cdap-api/src/main/java/co/cask/cdap/api/flow/FlowletDefinition.java

```diff
@@ -191,6 +203,27 @@ public int getInstances() {
   }
 
   /**
+   * @return Map of Stream name and {@link StreamSpecification} created in this Flowlet.
+   */
+  public Map<String, StreamSpecification> getStreams() {
```

Comment:
These are required because in ``DefaultFlowConfigurer`` we use ``FlowletDefinition`` to create and get ``FlowletSpecification`` instead of directly using ``FlowletConfigurer``. Hence we had to add these methods to access the streams/datasets created inside that Flowlet.